rest/blip_api_attachment_test.go

```diff
@@ -379,7 +379,7 @@ func TestBlipPushPullNewAttachmentNoCommonAncestor(t *testing.T) {
 		bodyText := `{"greetings":[{"hi":"alice"}],"_attachments":{"hello.txt":{"data":"aGVsbG8gd29ybGQ="}}}`
 		rev := NewDocVersionFromFakeRev("2-abc")
 		// FIXME CBG-4400:  docID: doc1 was not found on the client - expecting to update doc based on parentVersion RevID: 2-abc
-		err := btcRunner.StoreRevOnClient(btc.id, docID, &rev, []byte(bodyText))
+		_, err := btcRunner.AddRev(btc.id, docID, &rev, []byte(bodyText))
```

Comment:
`StoreRevOnClient` was obsolete and was just a wrapper around `AddRev` - removed for simplicity.